tests/integration_tests/scheduler/bin/qsub

```diff
@@ -13,6 +13,8 @@ do
             ;;
         k)
             ;;
+        l)
```

Comment:
This is not needed for the tests to pass, but is handy to have here if you want to manually run a config file with MEMORY_PER_JOB included.